src/framework/storage/FileSystemGraphObjectStore/path.ts

```diff
@@ -1,17 +1,17 @@
 export type CollectionType = 'entities' | 'relationships';
 
-interface BuildStepCollectionFilePathInput {
-  step: string;
+interface BuildObjectCollectionFilePathInput {
+  storageDirectoryPath: string;
   collectionType: CollectionType;
   filename: string;
 }
 
-export function buildStepCollectionFilePath({
-  step,
+export function buildObjectCollectionFilePath({
+  storageDirectoryPath,
   collectionType,
   filename,
-}: BuildStepCollectionFilePathInput) {
-  return ['graph', step, collectionType, filename].join('/');
+}: BuildObjectCollectionFilePathInput) {
+  return ['graph', storageDirectoryPath, collectionType, filename].join('/');
```

Comment:
Ah, here's that line I wondered about. Why not `path.join()`?

Reply:
tbh I forgot that was a function. Will change in separate PR.